packages/plugin-network-breadcrumbs/test/network-breadcrumbs.test.js

```diff
@@ -179,6 +186,54 @@ describe('plugin: network breadcrumbs', () => {
     })
   })
 
+  it('should handle a fetch() request supplied with a Request object', (done) => {
+    const window = { XMLHttpRequest, fetch }
+
+    const client = new Client(VALID_NOTIFIER)
+    client.setOptions({ apiKey: 'aaaa-aaaa-aaaa-aaaa' })
+    client.configure()
+    client.use(plugin, () => [], window)
+
+    const request = new Request('/')
+
+    window.fetch(request, {}, false, 200).then(() => {
+      expect(client.breadcrumbs.length).toBe(1)
+      expect(client.breadcrumbs[0]).toEqual(jasmine.objectContaining({
+        type: 'request',
+        name: 'fetch() succeeded',
+        metaData: {
+          status: 200,
+          request: 'GET /'
+        }
+      }))
+      done()
+    })
+  })
+
+  it('should handle a fetch() request supplied with a Request object that has a method specified', (done) => {
+    const window = { XMLHttpRequest, fetch }
+
+    const client = new Client(VALID_NOTIFIER)
+    client.setOptions({ apiKey: 'aaaa-aaaa-aaaa-aaaa' })
+    client.configure()
+    client.use(plugin, () => [], window)
+
+    const request = new Request('/', { method: 'PUT' })
+
+    window.fetch(request, {}, false, 200).then(() => {
+      expect(client.breadcrumbs.length).toBe(1)
+      expect(client.breadcrumbs[0]).toEqual(jasmine.objectContaining({
+        type: 'request',
+        name: 'fetch() succeeded',
+        metaData: {
+          status: 200,
+          request: 'PUT /'
+        }
+      }))
+      done()
+    })
+  })
+
```

Comment:
Please can you add a test cases for the following:

- `fetch(null)`
- `fetch(new Request('/foo'), { method: 'PUT' })`